manifests/installation-script

```diff
@@ -9,6 +9,7 @@ webHookToken = jsonSelect(operatorSecret, "data.WEBHOOK_TOKEN");
 grafanaPassword = jsonSelect(operatorSecret, "data.GRAFANA_PASSWORD");
 azureAccountKey = jsonSelect(operatorSecret, "data.AZURE_ACCOUNT_KEY");
 postgresSecret = kubectl get secret -n devtroncd postgresql-postgresql;
+enableLegacyApi = jsonSelect(operatorConfigMap, "data.ENABLE_LEGACY_API");
```

Comment:
we should be introducing flags specific to k8s version so that in future we can support multiple version compatibility

Reply:
it will be more intuitive for users also